go/libraries/doltcore/env/config.go

```diff
@@ -38,6 +38,10 @@ const (
 
 	DoltEditor = "core.editor"
 
+	// todo: analogous Git param is "init.defaultBranch", but
+	//   Dolt config params are case-insensitive
+	InitBranchName = "init.default_branch"
```

Comment:
If we're case insensitive, perhaps we should just allow `init.defaultbranch`? That way people can still type `init.defaultBranch` and it'll match.

Reply:
good call